adguard-vpn.com/privacy.html.md

```diff
@@ -65,16 +65,17 @@ Using our software. When you choose to use our app, you will provide us your dat
 - For AdGuard VPN for iOS: <https://adguard-vpn.com/privacy/ios.html>
 - For AdGuard VPN Browser Extension: <https://adguard-vpn.com/privacy/extension.html>
 
-**Usage information**. When you use our VPN service, we need to make sure that you stay within the agreed-upon limits. Therefore, in the free version, we collect data on your traffic usage. Additionally, we collect information on the number of instances of our software that have an active VPN connection for both the free and paid versions. We receive this information from our VPN servers through which you establish a VPN connection.
+**Usage information**. When you use our VPN service, we need to make sure that you stay within the agreed-upon limits. Therefore, we collect data on your traffic usage. This is done for two cases: to prevent service abuse and to know when a user's subscription has expired. We store only the number of bytes used by the account, it is not associated with locations. Traffic usage data is stored for 90 days. Additionally, we collect information on the number of instances of our software that have an active VPN connection for both the free and paid versions. We receive this information from our VPN servers through which you establish a VPN connection.
```

Comment:
So basically we need to collect usage data and know when a user's subscription expires or is canceled for one purpose — to prevent service abuse. Do I understand correctly?

Reply:
My subscription example is about correctly enforcing the free version limits. Service abuse is a different thing. Here's an example: someone installs a CLI version and then routes 1000 devices through it. Or builds their own VPN over it. Or something like that.